drivers/hmis/app/models/hmis/filter/household_filter.rb

```diff
@@ -24,6 +23,7 @@ def with_statuses(scope)
         return scope if input.status.sort == ['ACTIVE', 'INCOMPLETE', 'EXITED'].sort
 
         ids = []
+        # FIXME
```

Comment:
More context would be helpful - seems like it's not broken, but it could use optimization?